src/sickle.ts

```diff
@@ -212,12 +213,13 @@ class Annotator {
 
   private visitTypeAlias(node: ts.TypeAliasDeclaration) {
     if (this.options.untyped) return;
+    // Write a Closure typedef, which involves an unused "var" declaration.
     this.emit('/** @typedef {');
     this.visit(node.type);
     this.emit('} */\n');
     this.emit('var ');
     this.emit(node.name.getText());
-    this.emit(';\n');
+    this.emit(': any;\n');
```

Comment:
Lets emit `void` type for this variable, so that users can't shoot themselves in the foot by using this generated value.